lms/djangoapps/discussion/django_comment_client/base/tests.py

```diff
@@ -82,6 +82,7 @@ def _set_mock_request_data(self, mock_request, data):
 
 
 @patch('openedx.core.djangoapps.django_comment_common.comment_client.utils.requests.request', autospec=True)
+@patch('lms.djangoapps.discussion.toggles.ENABLE_FORUM_V2.is_enabled', autospec=True)
```

Comment:
Why not just use `@override_waffle_flag(ENABLE_FORUM_V2, True)`

Reply:
Yup, that can be also used but the idea was to keep the extra patching/mocking method same.

Reply:
With that method you wouldn't need to pass around the mock, and eventually when forums v2 is the only one you can just drop the relevant code easily.

Reply:
we have created test files(those will come in next PR) that cover forum v2. So in case when we are only left with forum v2, we'll eventually remove this file completely.